x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/list_action_types.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { UserAtSpaceScenarios } from '../../scenarios';
+import { getUrlPrefix } from '../../../common/lib/space_test_utils';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function listActionTypesTests({ getService }: FtrProviderContext) {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+  describe('list_action_types', () => {
+    for (const scenario of UserAtSpaceScenarios) {
+      const { user, space } = scenario;
+      describe(scenario.id, () => {
+        it('should return 200 with list of action types containing defaults', async () => {
```

Comment:
Can we restrict access to the API as well using https://github.com/elastic/kibana/blob/2b4f2a259e02f2a788d374a88b036c0444c44ed3/x-pack/legacy/plugins/xpack_main/server/lib/feature_registry/feature_registry.ts#L33-L57?

Reply:
Fixed in https://github.com/elastic/kibana/pull/41389/commits/59495f35b504da77d7f063bba6dddb5319c79e09.